code/src/components/TasksCompleted.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
Good idea to have also this as its own component. Makes it easy to find exactly what you are looking for quick if you need to do updates/changes.